.github/workflows/issue-to-json.yml

```diff
@@ -33,27 +32,25 @@ jobs:
           issue-template: "add-new-config.yml"
           # This controls which property we use to key the file name
           hash-property-name: "url_abbreviation"
-      - name: Commit files
+      
+      - name: Generate token to authenticate as op-config-updates GitHub App
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.CONFIG_UPDATES_GH_APP_ID }}
+          private_key: ${{ secrets.CONFIG_UPDATES_GH_APP_PRIVATE_KEY }}
+      
+      - name: Commit and PR changes
+        env:
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
```

Comment:
Just for my understanding, we changed from using `peter-evans/create-pull-request` to generating the token using `tibdex/github-app-token@v2` and then manually committing and creating the PR so that the PR would be "authorized" and would merge as soon as the tests passed, right?